lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1,14 +1,25 @@
 {
+  "fetchTime": "2019-10-10T21:09:41.484Z",
```

Comment:
I wouldn't expect these artifact changes to be necessary, did you run into an issue where you needed them?